src/avatar/loading-dots/index.tsx

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import styles from './styles.css.js';
+import clsx from 'clsx';
+
+interface LoadingDotsProps {
+  color?: string;
+}
+
+export default function LoadingDots({ color }: LoadingDotsProps) {
+  return (
+    <div className={clsx(styles.root, { [styles['gen-ai']]: color === 'gen-ai' })}>
```

Comment:
What is the need in having conditional gen-ai style? I thought the loading dots work the same for all colors

Reply:
On top of animated dots, gen-ai background color also animates, you can see [here](https://d21d5uik3ws71m.cloudfront.net/components/ee0a7eadf7a10c26ed248036c485d2a19f1fb86d/dev-pages/index.html#/light/avatar/basic). The solid colors don't really make sense to be animated

Reply:
Make sense, I would add a code comment to make this clear